app/views/stages/_fields.html.erb

```diff
@@ -23,23 +23,42 @@
     </div>
   </div>
 
-  <div class="form-group">
-    <div class="col-sm-offset-2 col-sm-10">
-      <div class="checkbox">
-        <%= form.label :confirm do %>
-          <%= form.check_box :confirm %>
-          Confirm before deployment
-        <% end %>
+  <% if ENV['DEPLOY_GROUP_FEATURE'] %>
+    <div class="form-group">
+      <%= form.label :name, 'Deploy Groups', class: 'col-lg-2 control-label' %>
+      <div class="col-lg-4">
+        <%= form.collection_select(:deploy_groups, DeployGroup.all, :id, :name,
+                                   {
```

Comment:
@grosser Not sure what you mean? This is 2-space indentation, and I'm aligning the parameters to the method at the same level according to: https://github.com/bbatsov/ruby-style-guide#no-double-indent.